pkg/kubelet/kubelet_node_status.go

```diff
@@ -156,6 +157,32 @@ func (kl *Kubelet) tryRegisterWithApiServer(node *v1.Node) bool {
 	return false
 }
 
+//updateDefaultLabels will set the default labels on the node
+func (kl *Kubelet) updateDefaultLabels(initialNode, existingNode *v1.Node) (needsUpdate bool) {
```

Comment:
Please keep the code style consistent, no need to define the return variable here.